core/src/main/java/forklift/consumer/MessageRunnable.java

```diff
@@ -144,8 +144,8 @@ public void run() {
                     LifeCycleMonitors.call(ProcessStep.Complete, this);
                 }
             }
-            // Always log all errors
-            getErrors().stream().forEach(e -> log.error(e));
+            // Always log all non-null errors
+            getErrors().stream().filter(e -> e != null).forEach(e -> log.error(e));
```

Comment:
I'm wondering if it would be useful at this point to wrap the stacktrace into an info message so that it can be reviewed. Thoughts?